src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs

```diff
@@ -514,6 +539,20 @@ private void HandlePgrPointerEvent(PointerRoutedEventArgs e, Action<View, Pointe
 			return new Point(result.Value.X, result.Value.Y);
 		}
 
+		Point GetPosition(UIElement element, Microsoft.UI.Xaml.DragStartingEventArgs e)
+		{
+			var result = e.GetPosition(element);
+
+			return new Point(result.X, result.Y);
```

Comment:
we do this 4 times or more, can we get an extension method? Do we own point? Can we add a constructor or a cast  operator so that we simple do:
```suggestion
			return (Point) e.GetPosition(element);
```

Reply:
Added extension method.